hyppo/independence/mirf.py

```diff
@@ -285,3 +295,27 @@ def test_cutoff(self, x, z, y):
         X_permutedZ = np.hstack((x, permuted_Z))
         null = self.statistic(X_permutedZ, y)
         return observe_stat, null
+
+    def test_diff(self, x, z, y, reps=1000, workers=1):
+        XZ = np.hstack((x, z))
+        observe_stat, observe_pos = self.statistic(XZ, y, return_pos=True)
+
+        # Compute statistic for permuted sets
+        permuted_Z = np.random.permutation(z)
```

Comment:
Do you need a random seed to make this reproducible?

Reply:
I don't think so, do we?